.env

```diff
@@ -9,3 +9,5 @@ NX_FIREBASE_MEASUREMENT_ID=G-EC9MJ6CP8X
 
 NX_SANITY_COURSES_PROJECT_ID=rzeg7i8f
 NX_SANITY_COURSES_DATASET=production
+
+FIRESTORE_EMULATOR_HOST=localhost:5502
```

Comment:
I'd rather have this hard-coded in the tests for now. I know that seems strange, but I've got a bigger plan for how we can automate this going forward.